core/iwasm/compilation/aot_llvm_extra.cpp

```diff
@@ -358,3 +368,86 @@ aot_apply_llvm_new_pass_manager(AOTCompContext *comp_ctx, LLVMModuleRef module)
 
     MPM.run(*M, MAM);
 }
+
+#if WASM_ENABLE_DYNAMIC_PGO != 0
+/*TODO: maybe move below to a new file? */
+
+/* append !{!"function_entry_count", i64 NN} */
+void
+wasm_dpgo_set_function_entry_count(LLVMValueRef function, uint32_t count)
```

Comment:
Had better change all `uint32_t` to `uint32`, we usually use the latter inside core/iwasm